### PR TITLE
PARQUET-134 patch - Support file write mode

### PR DESCRIPTION
**Pull request by masokan**

### CODE REVIEW

File: parquet-hadoop/src/main/java/parquet/hadoop/ParquetFileWriter.java [hunk 1]
```diff
@@ -66,6 +66,10 @@
   public static final byte[] MAGIC = "PAR1".getBytes(Charset.forName("ASCII"));
   public static final int CURRENT_VERSION = 1;
 
+  // File creation modes
+  public static final int CREATE = 0;
+  public static final int OVERWRITE = 1;
```
Comment (julienledem) [inline parquet-hadoop/src/main/java/parquet/hadoop/ParquetFileWriter.java]: Please create an enum for this.

**Comment by julienledem** [comment]
Thanks for contributing. I made a comment above. Otherwise this looks good to me.

**Comment by rdblue** [comment]
This was merged as #111. Can someone close it?

**Comment by julienledem** [comment]
@masokan could you close this pull request as the change was merged as part of another one?
Thank you.

**Pull request closed without merging**